package exam;

import java.sql.SQLException;

import exceptions.InvalidCityException;
import exceptions.InvalidNameExeption;
import exceptions.InvalidPersonalNumberException;

public class Demo {

	public static void main(String[] args) throws SQLException, InvalidNameExeption, InvalidCityException, InvalidPersonalNumberException {
		
		TestDbConnection test = new TestDbConnection();
		
		Person rabotnik = new Person("Stamat Ivanov Stamatov", "0885858585" , City.Sofia, "84858489"){};
		Person rabotnik2 = new Person("Vasil Gocev Stoev", "0599558787" , City.Varna, "84858489"){};
		Person rabotnik3 = new Person("Stamat Ivanov Stamatov", "0885858585" , City.Veliko_Tarnovo, "84858489"){};
		Person rabotnik4 = new Person("Stamat Ivanov Stamatov", "0885858585" , City.Ruse, "84858489"){};
		test.addEmployee(rabotnik);
		
		test.getEmployee("Alex");
	}

}
